src/OrchardCore.Modules/OrchardCore.Search.Elasticsearch/Views/Admin/Index.cshtml

```diff
@@ -20,7 +20,8 @@
                     <input asp-for="Options.Search" class="form-control" placeholder="@T["Search"]" type="search" autofocus />
                 </div>
                 <div class="btn-group float-end">
-                    <a asp-route-action="Edit" class="btn btn-sm btn-secondary float-end create" role="button">@T["Add Index"]</a>
+                    <a asp-route-action="SyncSettings" title="@T["Synchronize content index settings with Lucene."]" class="btn btn-sm btn-secondary sync" role="button" data-url-af="RemoveUrl UnsafeUrl" data-title=@T["Synchronize Content Index Settings"] data-message=@T["Are you sure you want to synchronize all Elasticsearch content index settings with Lucene ones?"]>@T["Synchronize Settings"]</a>
```

Comment:
Shouldn't this only be visible if Lucene is enabled?

Reply:
No, the Lucene ContentIndexSettings are persisted in the Content Definitions so we only copy them over to the Elastic ones. The Lucene module doesn't need to be enabled for this to work.

Reply:
I see, though there's nothing to sync if you never enabled the module.